src/components/navBar.tsx

```diff
@@ -53,6 +81,12 @@ const NavBar = ({}) => {
                     })}
                 </TabContainer>
             </div>
+            <div>
```

Comment:
give this div some width, w-2/3, or something. This will help you get a wider search bar